app/src/components/utils.ts

```diff
@@ -1,9 +1,21 @@
+import config from 'config';
 import { existsSync, readFileSync } from 'fs';
 import { join } from 'path';
 
 import { getLogger } from './log';
+import { ChefsFormConfig, ChefsFormConfigData } from '../types/ChefsFormConfig';
```

Comment:
this may sound like a lazy excuse.. but i would be tempted to just use more highlevel types like `object`, `array`, etc. where its safe to do that. especially while this app and the data model is still evolving